README.md

```diff
@@ -33,7 +33,10 @@ Feel free to make an issue to discuss amending the scope.
 
 ## Learning
 
-- The [PureScript Book](https://leanpub.com/purescript/read) is the recommended approach to learning the language, since it covers more material in greater depth. However, it is not updated yet for the 0.12 version of the compiler.
+- The [PureScript Book](https://leanpub.com/purescript/read) is the recommended approach to learning the language, since it covers more material in greater depth. However, it covers `0.11.7` and is not updated yet for the `0.12.x` version of the compiler. Thus, one should be aware of the following materials when reading through the book:
+    - See [dwhitney's fork of the book's exercises](https://github.com/dwhitney/purescript-book) and the other materials in this section for supplemental reading.
```

Comment:
It's not mentioned here that the usefulness of dwhitney's fork is that the code samples have been updated to work with 0.12.x version of the compiler -- I think it probably should. I could add a commit on top of this later, too, if you don't want to change it before merging.